Shared/Common/Structs/Environment.swift

```diff
@@ -189,11 +189,12 @@ public class Environment {
 
     /// Wrapper around CMPedometeer
     public struct Pedometer {
+        private let underlyingPedometer = CMPedometer()
```

Comment:
this is exactly the same as the CMMotionActivityManager above, not sure why i didn't do it here